src/components/profile/ProfileService.js

```diff
@@ -161,11 +161,11 @@ goog.require('ga_urlutils_service');
             params.headers = {'Content-Type':
                 'application/x-www-form-urlencoded'};
           }
-
+          var defaultElevationModel = gaGlobalOptions.defaultElevationModel;
           $http(params).success(function(data, status) {
             // When all the geometry is outside switzerland
             if (data.length == 0) {
-              data = [{alts: {COMB: 0}, dist: 0}];
+              data = [{alts: {defaultElevationModel: 0}, dist: 0}];
```

Comment:
This can't work, currently that set the variable ` alts['defaultElevationModel']` to 0. But whatever, you can set alts to an empty object because the 0 value is set here: https://github.com/ioda-net/mf-geoadmin3/blob/variables_elevation_models/src/components/profile/ProfileService.js#L99